OptionSelector/CQDetailTableViewController.h

```diff
@@ -0,0 +1,15 @@
+//
+//  CQDetailTableViewController.h
+//  Unit1WeekFourHomework
+//
+//  Created by Jackie Meggesto on 8/12/15.
+//  Copyright (c) 2015 Jackie Meggesto. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "CQCategory.h"
+
+@interface CQDetailTableViewController : UITableViewController
+@property (strong, nonatomic) CQCategory *category;
```

Comment:
Style: Object properties are `strong` by default, so you can remove `strong` from the attribute list here.